RecoParticleFlow/PFClusterProducer/python/particleFlowRecHitOOTECAL_cff.py

```diff
@@ -0,0 +1,6 @@
+import FWCore.ParameterSet.Config as cms
+from RecoParticleFlow.PFClusterProducer.particleFlowRecHitECAL_cfi import *
+
+particleFlowOOTRecHitECAL = particleFlowRecHitECAL.clone()
+particleFlowOOTRecHitECAL.producers[0].qualityTests[1].timingCleaning = cms.bool(False)
+particleFlowOOTRecHitECAL.producers[1].qualityTests[1].timingCleaning = cms.bool(False)
```

Comment:
so something like:
particleFlowOOTRecHitECAL.producers[0].qualityTests[1].timingCleaning = False?

I agree accessing by index is not ideal... producers[0] = EB, producers[1] = EE, so I guess if we move to a unified EBEE clustering, this would be a problem.